functorch/_src/aot_autograd.py

```diff
@@ -194,15 +196,19 @@ def forward(ctx, *flat_tensor_args):
                 compiled_bw = bw_compiler(bw_module, bw_args)
             else:
                 fw_outs = normalize_as_list(compiled_fw(*flat_tensor_args))
+            torch._C._jit_set_autocast_mode(old_jit_autocast_flag)
             ctx.save_for_backward(*fw_outs[num_outs:])
             return tuple(fw_outs[0:num_outs])
 
         @staticmethod
         @disable_torchdynamo
         def backward(ctx, *flat_args):
+            # Disable the JIT Autocast flag to prevent re-autocasting of jitted graph.
```

Comment:
Add a note that this is a hack, to be removed upon resolution of the Torchscript stuff (link to an issue or PR preferably.)